src/store/registration/saga.ts

```diff
@@ -222,22 +223,24 @@ export function* saga() {
   yield updateProfilePage();
 
   // After successful registration
-  yield spawn(openFirstConversation);
+  yield spawn(openFirstConversationAfterChannelsLoaded);
```

Comment:
Should be able to remove this now since it's not specific to the registration flow any more, right?

Reply:
done, i also moved the function implementation from registration to login, since it is where it's being used now.

Reply:
Yeah, makes sense!